qw11qD/platform.py

```diff
@@ -12,64 +15,72 @@
 def create():
     """Lines A and D of QuantWare 21q-chip controlled with Quantum Machines.
 
-    Current status (check before using):
-    Line A (6 qubits) is NOT calibrated because signal is very noisy at low readout power,
-    possibly due to not using the TWPA pump.
-    Line D (5 qubits): calibrated with TWPA and latest status in:
+    Current status (check before using): Line A (6 qubits) is NOT
+    calibrated because signal is very noisy at low readout power,
+    possibly due to not using the TWPA pump. Line D (5 qubits):
+    calibrated with TWPA and latest status in:
     https://github.com/qiboteam/qibolab_platforms_qrc/pull/149
     """
     lo_map = {q: f"{q}/drive_lo" for q in ["D1", "D4", "D5"]}
     lo_map["D2"] = lo_map["D3"] = "D2D3/drive_lo"
 
     twpa_d = SGS100A(name="twpaD", address="192.168.0.33")
 
-    qubits = {}
-    for i in range(1, 6):
-        q = f"D{i}"
-        qubits[q] = Qubit(
-            name=q,
-            probe=IqChannel(
-                name=f"{q}/probe",
-                mixer=None,
-                lo=f"D/probe_lo",
-                acquisition=f"{q}/acquisition",
-            ),
-            acquisition=AcquireChannel(
-                name=f"{q}/acquisition", twpa_pump=twpa_d.name, probe=f"{q}/probe"
-            ),
-            drive=IqChannel(name=f"{q}/drive", mixer=None, lo=lo_map[q]),
-            flux=DcChannel(name=f"{q}/flux"),
+    qubits = {
+        f"D{i}": Qubit(
+            drive=f"D{i}/drive",
+            flux=f"D{i}/flux",
+            probe=f"D{i}/probe",
+            acquisition=f"D{i}/acquisition",
         )
+        for i in range(1, 6)
+    }
 
     # Connect logical channels to instrument channels (ports)
```

Comment:
This comments seems outdated now, I would change to
```suggestion
    # Create channels and connect to instrument ports
```
or just drop completely.

Reply:
Done in 3ffa187